aten/src/ATen/core/Tensor.h

```diff
@@ -640,6 +640,8 @@ class CAFFE2_API Tensor {
   Tensor dequantize() const;
   double q_scale() const;
   int64_t q_zero_point() const;
+  Tensor q_scales() const;
```

Comment:
Lets call it something more distinctive: q_per_axis_scale and q_per_axis_zero_point? The current names differ by one letter and are unclear

Reply:
should be per_channel I think